### PR TITLE
changes to create a statefull localLoginService

### PR DESCRIPTION
**Pull request by JohnGorter**
Added code to give the LocalLoginService a statefull implementation. This gives the opportunity to add state-dependent widget to be developed and tested in the flutter_user userstory.

### CODE REVIEW

File: packages/flutter_login_service/lib/src/login_service.dart [hunk 1]
```diff
@@ -34,12 +35,22 @@ abstract class LoginServiceInterface {
   /// [logout] is used to log out the currently logged in user.
   Future<bool> logout(BuildContext context);
 
+  Future<bool> isLoggedId();
```
Comment (freekvandeven) [inline packages/flutter_login_service/lib/src/login_service.dart:38]: You made a typo here

**Review by freekvandeven — CHANGES_REQUESTED** [review]
need to update the changelog also, and if you update one of the packages of the login component you should update the other 2 aswell with a new version number (because it is quite hard to otherwise to keep this monorepo organized)

File: packages/flutter_login_service/lib/src/login_service.dart (hunk 1 above)
Comment (JohnGorter) [reply packages/flutter_login_service/lib/src/login_service.dart]: Sharp! Thanks

**Comment by freekvandeven** [comment]
the commit is still not renamed, You can use git commit --amend and then retype the commit message @JohnGorter

**Review by freekvandeven — APPROVED** [review]

**Comment by JohnGorter** [comment]
jeeej!

**Pull request merged**